hermetic_build/library_generation/DEVELOPMENT.md

```diff
@@ -21,9 +21,9 @@ code declared in a "golden branch" of the repo.
 It requires docker and python 3.x to be installed.
 
 ```
+python -m pip install --require-hashes -r requirements.txt
 python -m pip install .
```

Comment:
The development guide shows commands to be run from the `sdk-platform-java/library_generation` folder. However this command now must be run from `sdk-platform-java/hermetic_build/library_generation` (or change `pip install .` to `pip install library_generation` and run it from `//hermetic_build`. We can modify [this note](https://github.com/googleapis/sdk-platform-java/blob/d5e74d9b3cf38f8aef78f1990d94778a255a5d59/library_generation/DEVELOPMENT.md?plain=1#L2) to point to `hermetic_build`. 

With this, then we could correct this to

```suggestion
python -m pip install library_generation
```

Reply:
Done.